MediaRecorder.html

```diff
@@ -775,97 +775,101 @@
-                    <th>Type</th>
-                    <th>Nullable</th>
-                    <th>Optional</th>
-                    <th>Description</th>
-                  </tr>
-                  <tr>
-                    <td class="prmName">type</td>
-                    <td class="prmType"><code>DOMString</code></td>
-                    <td class="prmNullFalse"><span role="img" aria-label=
-                    "False">&#10008;</span></td>
-                    <td class="prmOptFalse"><span role="img" aria-label=
-                    "False">&#10008;</span></td>
-                    <td class="prmDesc"></td>
-                  </tr>
-                  <tr>
-                    <td class="prmName">eventInitDict</td>
-                    <td class="prmType"><code>ErrorEventInit</code></td>
-                    <td class="prmNullFalse"><span role="img" aria-label=
-                    "False">&#10008;</span></td>
-                    <td class="prmOptFalse"><span role="img" aria-label=
-                    "False">&#10008;</span></td>
-                    <td class="prmDesc"></td>
-                  </tr>
-                </tbody>
-              </table>
-            </dd>
-          </dl>
-        </section>
-        <section>
-          <h2>Attributes</h2>
-          <dl data-link-for="ErrorEvent" data-dfn-for="ErrorEvent" class=
-          "attributes">
-            <dt><dfn><code>error</code></dfn> of type <span class=
-            "idlAttrType"><a>Error</a></span>, readonly , nullable</dt>
-            <dd>If the event was raised because of an error, this attribute MAY
-            be set to that error object.</dd>
-          </dl>
-        </section>
-      </div>
+      specified, the <code><a data-link-for="MediaRecorderErrorEvent">error
+      </a></code> attribute defaults to null.</p>
+
       <div>
-        <pre class="idl">
-          dictionary ErrorEventInit : EventInit {
-            Error? error = null;
-          };
-        </pre>
-        <section>
-          <h2>Dictionary <a class="idlType">ErrorEventInit</a> Members</h2>
-          <dl data-link-for="ErrorEventInit" data-dfn-for="ErrorEventInit"
-          class="dictionary-members">
-            <dt><dfn><code>error</code></dfn> of type <span class=
-            "idlMemberType"><a>Error</a></span>, nullable, defaulting to
-            <code>null</code></dt>
-            <dd>If the event was raised because of an error, this attribute MAY
-            be set to that error object.</dd>
-          </dl>
-        </section>
-      </div>
+      <pre class="idl">
+        dictionary MediaRecorderErrorEventInit : EventInit {
+          required DOMException error;
+        };
+
+        [Exposed=Window, Constructor (DOMString type, MediaRecorderErrorEventInit eventInitDict)]
+        interface MediaRecorderErrorEvent : Event {
+          [SameObject] readonly attribute DOMException? error;
+        };
+      </pre>
+      <section>
+        <h2>Constructors</h2>
```

Comment:
Removed the table and added the link you mention, but left the constructor in the idl.  Or should I remove all references to the ctor?

The two examples I found: https://www.w3.org/TR/mediacapture-streams/#properties-of-overconstrainederror-instances and https://w3c.github.io/clipboard-apis/#clipboard-event-interfaces both explicit the constructor in the idl.